includes/REST_API/Stories_Controller.php

```diff
@@ -122,6 +122,27 @@ public function prepare_item_for_response( $post, $request ) {
 		return apply_filters( "rest_prepare_{$this->post_type}", $response, $post, $request );
 	}
 
+	/**
+	 * Updates a single post.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function update_item( $request ) {
+		$response = parent::update_item( $request );
+		if ( ! is_wp_error( $response ) ) {
+			// If publisher logo is set, let's assign that.
+			$publisher_logo_id = $request->get_param( 'publisher_logo' );
+			if ( $publisher_logo_id ) {
+				// @todo This option can keep track of all available publisher logo IDs in the future, thus the array.
+				$publisher_logo_settings           = get_option( self::PUBLISHER_SETTINGS_OPTION, [] );
```

Comment:
@spacedmonkey @swissspidy Added this instead of using the custom logo now.
Using this here as an array since based on Figma, the user can store more than one images as possible publisher logos and then choose among those. Didn't seem like having a separate category for this would make sense so perhaps we can store the logo options like this in an array, e.g. `['ids' => [1,2,3...], 'active' => 1]`, thoughts?

Reply:
This setting should have an array of allow ids and the picked logo should be stored in post meta on the story. Is that what you mean?

Reply:
What if that image is removed from the media library, should be remove it from the array and fallback?

Reply:
I'm actually a bit confused now about if the publisher logo should be global or not.

@pbakaus Could you chime in here? Based [on the document](https://docs.google.com/document/d/1Xs4uOZ3KCNlTMdjDT6q7cvTUpA1xk-kzrnVLQALltQk/edit#), it seems like the publisher logo is a global setting.

Could you confirm if each story can have its own publisher logo or is the publisher logo a global setting (stories sharing one logo)?

@spacedmonkey Note that having multiple publisher logos will not be resolved within this PR, within this PR just one ID will be used for the relevant story, however, I want to set up the option or meta or setting so that it could be later used [with multiple publisher logos](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=6123%3A2310) without having to do a migration.

Reply:
The publisher logo is global. On Figma, you'll see that it is connected to a global Settings page for the plugin. /cc @samitron7